packages/components/src/resizable-box/style.scss

```diff
@@ -1,16 +1,12 @@
 .components-resizable-box__handle {
 	display: none;
+	width: $resize-handler-container-size;
+	height: $resize-handler-container-size;
 
 	// Show the resize handle when selected.
 	.components-resizable-box__container.is-selected & {
 		display: block;
 	}
-
-	// The handle is a pseudo-element and will sit inside this larger
-	// container size.
-	width: $resize-handler-container-size;
-	height: $resize-handler-container-size;
-	padding: $grid-size-small;
```

Comment:
Did we lose this padding? If so why and is it safe?

Reply:
Yeah, it's gone. The original handle consisted of a `div` that was just a tad bigger than the blue dot and the padding made sure it's centered inside it. 

Now, I have changed a lot how the handle wrapper is sized - it is now as big as the whole image on one side and I've replaced the padding centering approach with more suiting absolute position for the blue dot.